SConstruct

```diff
@@ -13,7 +13,12 @@ import methods
 import gles_builders
 from platform_methods import run_in_subprocess
 
-# scan possible build platforms
+# We now require Python 3.5+
+if not (sys.version_info.major == 3 and sys.version_info.minor >= 5):
+    print("Godot's buildsystem requires Python 3.5 or later. Be sure to also install SCons for Python 3.")
+    Exit(1)
+
```

Comment:
I think [`EnsurePythonVersion(3, 5)`](https://scons.org/doc/production/HTML/scons-man.html#f-EnsurePythonVersion) can be used instead provided by SCons next to `EnsureSConsVersion`.

Reply:
Thanks, done.